src/app/pages/datasets/components/dataset-management/dataset-management.component.html

```diff
@@ -65,16 +65,11 @@
         class="dataset-tree-wrapper"
       >
         <div class="dataset-tree-inner">
-          <ix-tree-virtual-scroll-view
```

Comment:
Using `ix-tree-virtual-scroll-view` here is intentional, as removing it will cause performance issues when the system has multiple datasets.

Reply:
@denysbutenko Didn't we manage to implement both virtual scrolling and full page scrolling or am I misremembering things?

Reply:
> @denysbutenko Didn't we manage to implement both virtual scrolling and full page scrolling or am I misremembering things?

@undsoft I don't remember this tbh. I think we have skipped backporting `ix-tree-virtual-scroll-view` for Bluefin. It may cause different scrolling behavior between Bluefin and Cobia versions.